packages/react-native/ReactAndroid/src/main/java/com/facebook/react/ReactRootView.java

```diff
@@ -632,7 +632,7 @@ public void setEventListener(@Nullable ReactRootViewEventListener eventListener)
 
   @Override
   public String getJSModuleName() {
-    return Assertions.assertNotNull(mJSModuleName);
+    return mJSModuleName;
```

Comment:
I thought we were adding back the assertion. Can you please update this?

Reply:
haha it looks like the original version is more correct. sorry i totally misunderstood what you said before.
updated and please let me know if there are any other feedback.